web/src/components/Diagram/components/TimelineChart.tsx

```diff
@@ -1,26 +1,26 @@
 import {useCallback, useEffect, useMemo, useRef} from 'react';
 import * as d3 from 'd3';
-import TraceAnalyticsService from '../../../services/Analytics/TraceAnalytics.service';
-import {IDiagramProps} from '../Diagram';
-import {TSpan} from '../../../types/Span.types';
-import {getNotchColor} from '../../TraceNode/TraceNode.styled';
+import TraceAnalyticsService from 'services/Analytics/TraceAnalytics.service';
+import {TSpan} from 'types/Span.types';
+import {getNotchColor} from 'components/TraceNode/TraceNode.styled';
+import {IDiagramComponentProps} from '../Diagram';
 import * as S from './TimelineChart.styled';
 
 const {onTimelineSpanClick} = TraceAnalyticsService;
 
 const barHeight = 54;
 
-export const TimelineChart = ({affectedSpans, trace, selectedSpan, onSelectSpan}: IDiagramProps) => {
+export const TimelineChart = ({affectedSpans, spanList, selectedSpan, onSelectSpan}: IDiagramComponentProps) => {
```

Comment:
Do we want to apply the "matched" style to spans in this `Timeline` chart?

Reply:
I'll ask Olly as I don't know what would be the best way of doing it for the timeline view and add it on a follow up PR 😄